website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -79,6 +79,41 @@ so no changes are needed.
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint)
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint_service](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service)
 
+## Migration from Cloud Provider Snapshot Backup Policy to Cloud Provider Backup Schedule
+
+To upgrade from v0.9.1 to v1.0.0 with the existing resources `mongodbatlas_cloud_provider_snapshot_backup_policy`
+perform the following steps
```

Comment:
Shouldn't we also note that the cluster parameter is changing as well?

Reply:
@coderGo93 this still isn't addressed in the steps.   It simply notes that " **IMPORTANT** In resource `mongodbatlas_cluster` the parameter `provider_backup_enabled` has been deprecated, use `cloud_backup` instead."  Should we not recommend as part of the transition change from the old to the new associated cluster parameter?   Will both work okay if they use  `provider_backup_enabled` with `mongodbatlas_cloud_backup_schedule`?